doc/_templates/autosummary.rst

```diff
@@ -6,11 +6,14 @@
 .. auto{{ objtype }}:: {{ objname }}
 
 {% if objtype in ['class', 'method', 'function'] %}
+{% if objname not in ['AxesGrid', 'Scalable', 'HostAxes', 'FloatingAxes',
+                      'ParasiteAxesAuxTrans', 'ParasiteAxes'] %}
```

Comment:
I assume the plan is to remove these after their docstrings are fixed?  Perhaps leave a comment to that effect.

Reply:
This relates to sphinx-gallery/sphinx-gallery#365; so yes, once that is fixed, one can remove.